packages/nameguard-react/src/components/Search/SearchEmptyState.tsx

```diff
@@ -26,6 +26,7 @@ const examples = [
   "unknоwn.eth",
   "john🇺🇸",
   "7️⃣7️⃣7️⃣.eth",
+  "testingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtestingtesting.eth",
 ].map((name) => buildENSName(name));
 
 export const SearchEmptyState = () => {
```

Comment:
@notrab Hey could you please investigate an issue where we now are able to fully scroll completely off the end of the right side of the search empty state?

![CleanShot 2024-08-19 at 18 42 36](https://github.com/user-attachments/assets/462c3f4f-297e-4cb1-95d7-22942723fccc)

Above is a screenshot from my local environment after I updated this example uninspected name.

Reply:
I've created a separate story to track this goal: https://app.shortcut.com/ps-web3/story/25668/fix-horizontal-scrolling-bug